Dockerfile

```diff
@@ -21,9 +22,13 @@ RUN git clone ${POLY_GIT} ${POLY_DIR} && \
 RUN git clone ${OVEN_GIT} ${OVEN_DIR} && \
     cd ${OVEN_DIR} && \
     sha1sum worker.sml > cakeml-token && \
-    uname -norm > name && \
     polyc worker.sml -o worker
 
+RUN useradd -u 1050 -m oven && chown -R oven:oven /oven
+RUN dnf install -y psmisc diffutils
```

Comment:
If you want to spend an extra week testing this i'm happy to give you a Dockerfile that follows best practices for command ordering and bundling.

Reply:
What sort of test is being done to this container?

Reply:
Basically I want to see at least one green build on https://cakeml.org/regression.cgi/ with the containerized setup.  I'm a little bit frustrated because I took over the infra on Friday the 18th and all I've achieved so far is to clutter up the list with aborted builds.